src/test/java/org/sagebionetworks/web/unitserver/ChallengeClientImplTest.java

```diff
@@ -154,10 +153,7 @@ public void before() throws SynapseException, JSONObjectAdapterException {
 
 		mockEvaluation = Mockito.mock(Evaluation.class);
 		when(mockSynapse.getEvaluation(anyString())).thenReturn(mockEvaluation);
-		mockUserSessionData = Mockito.mock(UserSessionData.class);
 		mockUserProfile = Mockito.mock(UserProfile.class);
-		when(mockSynapse.getUserSessionData()).thenReturn(mockUserSessionData);
```

Comment:
These calls are going to be removed from the java client, so let's clean them up now.